ios/Sources/HealthPluginPlugin/HealthPlugin.swift

```diff
@@ -102,8 +108,67 @@ public class HealthPlugin: CAPPlugin, CAPBridgedPlugin {
             return nil
         }
     }
-    
-    
+
+    @objc func querySamples(_ call: CAPPluginCall) {
+        guard let startDateString = call.getString("startDate"),
+              let endDateString = call.getString("endDate"),
+              let dataTypeString = call.getString("dataType"),
+              let startDate = self.isoDateFormatter.date(from: startDateString),
+              let endDate = self.isoDateFormatter.date(from: endDateString) else {
+            call.reject("Invalid parameters")
+            return
+        }
+
+        guard let dataType = aggregateTypeToHKQuantityType(dataTypeString) else {
```

Comment:
When I try to query samples for  resting HR, HRV or stand time, I get invalid data type. It looks like the `aggregateTypeToHKQuantityType()` method was not changed to support these data types. Maybe there's some more work here...